### PR TITLE
create-react-app on create container

### PR DESCRIPTION
**Pull request by WillianAgostini**
Related to issue https://github.com/github/codespaces-react/issues/3.
With these changes the create-react-app script is executed when the container is started.

### CODE REVIEW

**Review by bdmac — COMMENTED** [review]
👋 Thanks for the PR! I'm a little confused as this looks like it deletes a good chunk of the template?

**Comment by WillianAgostini** [comment]
> 👋 Thanks for the PR! I'm a little confused as this looks like it deletes a good chunk of the template?

Yes, it deletes the current project because the project is always created by the create-react-app script, always bringing the most stable version of the react app. The idea here is not to need to update the template from time to time.

**Comment by siddhartt-folio** [comment]
commit

**Review by siddhartt-folio — APPROVED** [review]

**Comment by bdmac** [comment]
I believe that would interact poorly with our codespace prebuilds on this project. Currently while it's a frustrating burden to update the template periodically it provides the best possible user experience because they don't have to wait for any dependencies to be installed or anything to be generated by `create-react-app`. This would be net-better for us, the maintainers, but worse for our users.

**Comment by WillianAgostini** [comment]
It is a good observation point. I took into account the user's requirement to always start with the most up-to-date source code version, in the same way that the machine is configured before use.

**Comment by bdmac** [comment]
For sure that is also important. I think what that means though (unfortunately for us haha) is that we would just need to periodically update the packages and possibly regenerate the app so the prebuild has everything ready to go for folks.

**Comment by hardikary** [comment]
yo who fuck are you

File: .gitignore [hunk 1]
```diff
@@ -1,127 +0,0 @@
-# Logs
```
Comment (ponury1990) [inline .gitignore:1]: ],
	"main": "./out/extension.js",
	"contributes": {},

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "hostRequirements": {
     "cpus": 4
   },
+  "onCreateCommand": "bash .devcontainer/create-project.sh",
```
Comment (ponury1990) [inline .devcontainer/devcontainer.json:6]: * .log
npm-debug.log *
przędza-debug.log *
dziennik błędów przędzy *
lerna-debug.log *

File: src/setupTests.js [hunk 3]
```diff
@@ -1,5 +0,0 @@
-// jest-dom adds custom jest matchers for asserting on DOM nodes.
-// allows you to do things like:
-// expect(element).toHaveTextContent(/react/i)
-// learn more: https://github.com/testing-library/jest-dom
```
Comment (ponury1990) [inline src/setupTests.js:4]: "onCreateCommand" : "bash .devcontainer/create-project.sh"

**Review by ponury1990 — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (ponury1990) [inline .devcontainer/devcontainer.json:6]: "portyAtrybuty" : {
    "3000" : {
      "etykieta" : "Aplikacja" ,
      "onAutoForward" : "openPreview"
    }
  } ,
  "forwardPorts" : [ 3000 ]
}

File: .gitignore (hunk 1 above)
Comment (hardikary) [reply .gitignore:1]: you can research on GitHub marketplace for extension for js

**Pull request closed without merging**